Funcky.Money/MoneyEvaluationContext.cs

```diff
@@ -60,14 +60,9 @@ private Builder(Option<Currency> currency, Option<decimal> distributionUnit, Opt
         }
 
         public MoneyEvaluationContext Build()
-        {
-            if (CompatibleRounding().Match(none: false, some: Negate))
-            {
-                throw new IncompatibleRoundingException($"The roundingStrategy {_roundingStrategy} is incompatible with the smallest possible distribution unit {_distributionUnit}.");
-            }
-
-            return CreateContext();
-        }
+            => CompatibleRounding().Match(none: false, some: Not<bool>(Identity))
```

Comment:
You could use

`is [false]`

that would work, right?

Reply:
Otherwise, wouldn't `Not<bool>` be sufficient? Is `Identity` needed?

Reply:
I would need to use C# 11 and Funcky 3.3. 
I would increase the C-Sharp version, but since I do not want to increase the dependency of Funcky to 3.3 at the moment I cannot use that featuer.[](url)